src/frontend/screens/Login/components/Runner/index.tsx

```diff
@@ -15,21 +15,15 @@ interface RunnerProps {
   logoutAction: () => any
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   alternativeLoginAction?: () => any
+  buttonText: string
+  disabled: boolean
 }
 
 export default function Runner(props: RunnerProps) {
   const maxNameLength = 20
   const { t } = useTranslation()
-  const [isLoggingOut, setIsLoggingOut] = React.useState(false)
-
-  let buttonText = ''
-  if (props.class === 'epic') {
-    buttonText = t('login.epic', 'Epic Games Login')
-  } else if (props.class === 'nile') {
-    buttonText = t('login.amazon', 'Amazon Login')
-  } else {
-    buttonText = t('login.gog', 'GOG Login')
-  }
```

Comment:
this is extra in this PR, but I think this strings shouldn't be in this component but passed as props so I did a small refactor